### ISSUE TITLE
Accessibility Features for Selenium Testing

### ISSUE THREAD

**Issue by jonreding2010** [Magenic/JMAQS#366]
### Add
- [x]  Unit tests for creating files of scans for invalid accessibility on web sites in TXT format
- [x]  Unit tests for creating files of scans for invalid accessibility on web sites in HTML format
- [ ]  Add AddFixes content to HTML Reporter
- [ ]  Add to include images of HTML web element violation

### Review
- [ ] 

### Remove
- [ ] 

### Branches/PRs
Module PR:  #479

### Notes
Use https://github.com/dequelabs/axe-core-maven-html repository for this issue.
